pkg/placement/raft/logger.go

```diff
@@ -86,6 +86,10 @@ func (l *loggerAdapter) ResetNamed(name string) hclog.Logger { return l }
 
 func (l *loggerAdapter) SetLevel(level hclog.Level) {}
 
+func (l *loggerAdapter) GetLevel() hclog.Level {
+	return hclog.Info
```

Comment:
why is this always returning info?

Reply:
This logger is kind of a nop and info is the default level